### ISSUE TITLE
Contextual information missing for `leaflet-control-layers-list` when control is collapsed

### ISSUE THREAD

**Issue by danallsop** [ScottLogic/jumpstart-Leaflet#303]
> **Migrated from:** https://github.com/Leaflet/Leaflet/issues/7529
> **Original author:** danallsop
> **Original created:** 2021-03-28T15:04:09Z
---

## Motivation

The `<a class="leaflet-control-layers-toggle">`'s `title="Layers"` provides a summary (context) for the content that it controls (for screen reader users), however when the control's `collapsed` option is set to `false` the control is `display: none` and no summary is provided for the content.

## Proposed solution

When the layer control's `collapsed` option is set to `false` the [`leaflet-control-layers-list`](https://github.com/Leaflet/Leaflet/blob/02f204e782842fa32b3b8e2fbb8084121b74b8f1/src/control/Control.Layers.js#L184) element should have `aria-label="Layers"`.

## Additional context

- MDN web docs: [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label)
- Relates to <span title="Web Content Accessibility Guidelines">WCAG 2.1</span> <span title="Success Criteria">SC</span> [1.3.1 Info and Relationships](https://www.w3.org/TR/WCAG21/#info-and-relationships).

**Comment by danallsop** [comment]
## Comments (4 total)

---
> **Comment by:** danallsop **on** 2023-09-03T16:58:07Z

Ah, I've found [#7526](https://github.com/Leaflet/Leaflet/issues/7526) (Markers that bind to popups should convey the `aria-expanded` state of the popup), so if I've misunderstood things my apologies!
---
> **Comment by:** danallsop **on** 2023-10-05T21:03:45Z

I added a command inside when if not collapsed in Control.LayersSpec.js
map.ariaLabel="Layers"
`
		it('when not collapsed', () => {
			new Control.Layers(null, null, {collapsed: false}).addTo(map);/
			expect(map._container.querySelector('.leaflet-control-layers-toggle')).to.be.ok;
			map._container.ariaLabel="Layers"
		});
`
---
> **Comment by:** danallsop **on** 2023-10-06T16:31:05Z

danallsop if you want to contribute please make a Pull Request
---
> **Comment by:** danallsop **on** 2023-10-07T10:26:52Z

I would like to make a pull request
